README.md

```diff
@@ -14,6 +14,82 @@ This package defines some tests which invoke `perf_test` from Apex.AI's [perform
   - rmw_fastrtps_dynamic_cpp
   - rmw_opensplice_cpp
 
+### Test 1 - Performance Test  (APEX.AI)
+
+In this test we are testing the Performance Test provided by APEX.AI. Right now we have [our own fork](https://github.com/ros2/performance_test) because there are some pending pull requests in the official gitlab repository.
+
+In this test we are measurement:
+ - Average round-trip time
+ - CPU usage (provided by APEX.AI tool)
+ - Sent/Received packets per second
+ - Total lost packets
+ - Max resident set size
+
+We are generating two plots per measurement
+ - [One per per-build](http://build.ros2.org/view/Eci/job/Eci__nightly-performance_ubuntu_bionic_amd64/lastBuild/)
+ - [Other over time](http://build.ros2.org/view/Eci/job/Eci__nightly-performance_ubuntu_bionic_amd64/plot/)
+
+**The test only measure the latency between the same rmw implementation**
+
+| Publisher/Subscriber     | rmw_fastrtps_cpp         | rmw_opensplice_cpp       | rmw_cyclonedds_cpp       | rmw_fastrtps_dynamic_cpp | rmw_connext_cpp          |
+|--------------------------|--------------------------|--------------------------|--------------------------|--------------------------|--------------------------|
+| rmw_fastrtps_cpp         | :heavy_check_mark:       | :heavy_multiplication_x: | :heavy_multiplication_x: | :heavy_multiplication_x: | :heavy_multiplication_x: |
+| rmw_opensplice_cpp       | :heavy_multiplication_x: | :heavy_check_mark:       | :heavy_multiplication_x: | :heavy_multiplication_x: | :heavy_multiplication_x: |
+| rmw_cyclonedds_cpp       | :heavy_multiplication_x: | :heavy_multiplication_x: | :heavy_check_mark:       | :heavy_multiplication_x: | :heavy_multiplication_x: |
+| rmw_fastrtps_dynamic_cpp | :heavy_multiplication_x: | :heavy_multiplication_x: | :heavy_multiplication_x: | :heavy_check_mark:       | :heavy_multiplication_x: |
+| rmw_connext_cpp          | :heavy_multiplication_x: | :heavy_multiplication_x: | :heavy_multiplication_x: | :heavy_multiplication_x: | :heavy_check_mark:       |
+
+## Test 2 - Simple pub/sub
+
+In this case we are testing one publisher and one subscriber **in different processes** sending a 1kArray at 5Hz. This will allow us to
+ evaluate additional overhead caused by a single pub-sub topic and detect leaks related to this
+activity.
+
+We measurement for both publisher and subscriber:
+
+ - Average round trip
+ - CPU usage ( readed from the filesystem )
+ - Total lost packets
+ - Received/Sent packets
+ - Physical memory (Mb)
+ - Resident anonymous memory (Mb)
+ - Virtual memory (Mb)
```

Comment:
These measurements have units, while the rest don't. I don't really think it's important to state what the units are in this list, but if you decided to do it, please do it in all of the measurement lists.